app/services/claims/external_user_actions.rb

```diff
@@ -0,0 +1,15 @@
+module Claims
+  class ExternalUserActions
+    def self.all
+      Settings.claim_actions
+    end
+
+    def self.available_for(claim)
+      if claim.applicable_for_written_reasons?
+        Settings.claim_actions
+      else
+        [] << Settings.claim_actions[0]
```

Comment:
Is it worth removing the specific reason that is not applicable, i.e. Request written reasons from the existing array

```
Settings.claim_actions.reject{|action| action.match? 'written reasons')
```